test-app/src/components/app.tsx

```diff
@@ -9,7 +9,19 @@ import { useAuth } from '@common/use-auth';
 import { GlobalStyles } from '@components/global-styles';
 
 export const App: React.FC = () => {
-  const { authOptions, state, authResponse, appPrivateKey, handleSignOut } = useAuth();
+  const { authOptions, state, authResponse, appPrivateKey, handleSignOut, handleIsOnboarding } =
+    useAuth();
+
+  const handleMessage = (event: MessageEvent) => {
+    if (event.data.method === 'completeOnboarding') {
+      handleIsOnboarding(true);
+    }
```

Comment:
This seems strange to me, there hasn't been any other message type event watching we've pushed to app devs as their responsibility. Maybe this can be abstracted into a hook, like `useIsOnboarding` or something. I'll have to think more about this.

Reply:
We can definitely take it out of this PR for now, wasn't quite sure either. It will land the user back on the tab successfully with a reload of the tab so maybe that is all we do for now. I couldn't think of another solution for letting the app know the extension had been installed successfully and the user was back.